### ISSUE TITLE
--static option doesn't return highstate output

### ISSUE THREAD

**Issue by doesitblend** [saltstack/salt#44556]
### Description of Issue/Question
When using `--static` with a state call such as `salt \* state.highstate test=True --static --state-verbose=False` output falls back to `pprint` outputter. Unable to force the `highstate` outputter with `--out=highstate`. 

### Setup
Configure a master/minion with a state such as:
```
update_contact_letter:
  file.managed:
    - name: /var/www/html/form_letter.html
    - source: salt://testlocal/form_letter.html
    - template: jinja
    - mode: 644
```

Then run the state with `salt 'vagrant*' state.sls testlocal.mod_file test=True --static --state-verbose=False --out=highstate`. 

### Versions Report
```
Salt Version:
           Salt: 2017.7.2
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 1.5
      docker-py: Not Installed
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 0.9.1
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.6 (default, Oct 26 2016, 20:30:19)
   python-gnupg: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.0.1
           RAET: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: Ubuntu 14.04 trusty
         locale: UTF-8
        machine: x86_64
        release: 3.13.0-128-generic
         system: Linux
        version: Ubuntu 14.04 trusty
```

**Comment by doesitblend** [comment]
ZD-1987

**Comment by cachedout** [comment]
@rallytime Please see what would be involved with this fix and comment here. I need this by next Wednesday, please. Thanks.

**Comment by rallytime** [comment]
@cachedout I will take a look today.

**Comment by rallytime** [comment]
@cachedout and @doesitblend This should be fixed with #44714.
